dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -1 +1,5 @@
+- if @level.supports_markdown?
+  - content_for(:head) do
+    %script{src: webpack_asset_path('js/levels/editors/_dsl.js')}
```

Comment:
this is fine to get unblocked, but, ideally we want to keep these kind of conditional partials simple, with a haml file unconditionally including the corresponding js file when possible. in this particular case, the js file could simply recover gracefully if the DOM element it is looking for doesn't load.

Reply:
It looks like the only DSL defined level that does not have `supports_markdown?` is LevelGroup. I could just pull out Level Group as its own partial to solve this. I won't do it here but if you think thats the right move I can do it in a follow up